src/main/java/org/terasology/web/resources/base/Resource.java

```diff
@@ -15,11 +15,29 @@
  */
 package org.terasology.web.resources.base;
 
+/**
+ * Base interface for the resource implementation.
+ */
 public interface Resource {
 
+    /**
+     * Determine the method used to request a resource.
+     * @param methodName type of http request (GET, PUT, etc.).
```

Comment:
`GET`, `POST`, etc are HTTP **methods** (see the [RFC here](https://tools.ietf.org/html/rfc7231#page-22). An HTTP request is the whole message containing the method, the path, and request headers.